mods/cnc/rules/infantry.yaml

```diff
@@ -15,6 +15,7 @@ E1:
 	Armament:
 		Weapon: M16
 	AttackFrontal:
+		Voice: Move
```

Comment:
You're arguing 21 that duplicated `Voice: Move` definitions is less redundant than vs 1 duplicated voice definition.